authentication/templates/sysadmin/user_list.html

```diff
@@ -70,24 +66,29 @@ <h1>User Directory</h1>
         <thead>
           <tr>
             <th scope="col">Id</th>
+            <th scope="col">Type</th>
             <th scope="col">Username</th>
             <th scope="col">Email</th>
             <th scope="col">First Name</th>
             <th scope="col">Last Name</th>
             <th scope="col">Is Super User</th>
             <th scope="col">Is Active</th>
+            <th scope="col"></th>
           </tr>
         </thead>
         <tbody>
           {% for user in user_list %}
           <tr class="user-row" data-url="{% url 'authentication:user-detail-admin' user.id %}">
             <td> {{ user.id }}</td>
+            <td> {{ user.type }}</td>
             <td> {{ user.username }}</td>
             <td> {{ user.email }}</td>
             <td> {{ user.first_name }}</td>
             <td> {{ user.last_name }}</td>
             <td> {{ user.is_superuser }}</td>
             <td> {{ user.is_active }}</td>
+            <td> <a href="/auth/sysadmin/users/edit/{{user.type}}/{{user.id}}"><button
+                  class="btn btn-primary">Edit</button></a></td>
```

Comment:
I like the edit button. I had originally added edit functionality when clicking on the row. We should remove the edit click on the row so it doesn't cause confusion.